lib/tasks/annotate_rb.rake

```
@@ -0,0 +1,8 @@
+# This rake task was added by annotate_rb gem.
+
+# Can set `ANNOTATERB_SKIP_ON_DB_TASKS` to be anything to skip this
+if Rails.env.development? && ENV["ANNOTATERB_SKIP_ON_DB_TASKS"].nil?
```

rubocop (1.69.1)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)